content/developer/misc/i18n/localization.rst

```diff
@@ -303,123 +588,68 @@ To create a new fiscal position, simply use the ``account.fiscal.position.templa
         <field name="name">...</field>
 
         <!-- [Required] Set the CoA owning this fiscal position. -->
-        <field name="chart_template_id" ref="..."/>
+        <field name="chart_template_id" ref="l10n_XX_chart_template"/>
+
+        <!-- [Optional] Can be used to sort list of fiscal positions. The ones with lowest value comes first. Zero by default -->
+        <field name="sequence">10</field>
 
         <!-- [Optional] Add some additional notes. -->
         <field name="note">...</field>
-    </record>
 
-Adding the properties to my Chart of Accounts
-#############################################
+        <!-- [Optional] Apply automatically this fiscal position. False by default-->
+        <field name="auto_apply" eval="False"/>
 
-When the whole accounts are generated, you have the possibility to override the newly generated chart of accounts by adding some properties that correspond to default accounts used in certain situations.
-This must be done after the creation of accounts before each one must be linked to the chart of accounts.
+        <!-- [Optional] When set to True, it can be applied only if partner has VAT number. False by default -->
+        <field name="vat_required" eval="True"/>
```

Comment:
I'm not sure which version is better

JCO: it can **only** be applied when the partner has a VAT number. 
https://context.reverso.net/%D0%BF%D0%B5%D1%80%D0%B5%D0%B2%D0%BE%D0%B4/%D0%B0%D0%BD%D0%B3%D0%BB%D0%B8%D0%B9%D1%81%D0%BA%D0%B8%D0%B9-%D1%84%D1%80%D0%B0%D0%BD%D1%86%D1%83%D0%B7%D1%81%D0%BA%D0%B8%D0%B9/it+can+only

IEL:  it can be applied **only** if partner has VAT number.

https://context.reverso.net/%D0%BF%D0%B5%D1%80%D0%B5%D0%B2%D0%BE%D0%B4/%D0%B0%D0%BD%D0%B3%D0%BB%D0%B8%D0%B9%D1%81%D0%BA%D0%B8%D0%B9-%D1%84%D1%80%D0%B0%D0%BD%D1%86%D1%83%D0%B7%D1%81%D0%BA%D0%B8%D0%B9/only+if